gtsfm/evaluation/metrics_report.py

```diff
@@ -84,12 +110,22 @@ def create_plots_for_distributions(metrics_dict: Dict[str, Any]) -> str:
         if "histogram" in metric_value[metrics.SUMMARY_KEY]:
             histogram = metric_value[metrics.SUMMARY_KEY]["histogram"]
             fig.add_trace(
-                go.Bar(x=list(histogram.keys()), y=list(histogram.values()), name=metric_name), row=row, col=col
+                go.Bar(
+                    x=list(histogram.keys()),
+                    y=list(histogram.values()),
+                    name=metric_name,
+                ),
+                row=row,
+                col=col,
             )
         elif "quartiles" in metric_value[metrics.SUMMARY_KEY]:
             # If all values are available, use them to create box plot.
             if metrics.FULL_DATA_KEY in metric_value:
-                fig.add_trace(go.Box(y=metric_value[metrics.FULL_DATA_KEY], name=metric_name), row=row, col=col)
+                fig.add_trace(
+                    go.Box(y=metric_value[metrics.FULL_DATA_KEY], name=metric_name),
+                    row=row,
+                    col=col,
+                )
```

Comment:
probably needs to be reformatted with the 120 character limit?

Reply:
Reverted